docs/source/notes/autograd.rst

```diff
@@ -353,8 +353,8 @@ potential parallelism, it's important that we ensure thread safety on CPU with
 parallel backwards that share part/whole of the GraphTask.
 
 Custom Python ``autograd.function`` is automatically thread safe because of GIL.
-for built-in C++ Autograd Nodes(e.g. AccumulateGrad, CopySlices) and custom
-``autograd::Function``, the Autograd Engine uses thread mutex locking to protect
+For built-in C++ Autograd Nodes (e.g. AccumulateGrad, CopySlices) and custom
```

Comment:
nit: could you update the line above to make it `autograd.Function` as well?

Reply:
Sure :)